wiki/pages/higher-order-judgements.elf

```diff
@@ -81,11 +103,4 @@ The constants ``of-empty`` and ``of-app`` correspond to the informal inference r
 * [Reformulating languages to use hypothetical judgements](/wiki/reformulating-languages-to-use-hypothetical-judgements/)
 * [higher-order abstract syntax](/wiki/higher-order-abstract-syntax/)
 
-\{\{stub\}\} !}%
```

Comment:
Is removing this the right thing to do? I'd like to contribute more but I don't want to cause any issues.

Reply:
Yep - stuff like that is generally just artifacts from the MediaWiki conversion